app/Utils/Helpers.php

```diff
@@ -7,9 +7,9 @@
 class Helpers extends Nette\Object
 {
 
-	public static function dateTime(\DateTime $s)
+	public static function dateTime($s)
 	{
-		return $s->format('j. n. Y H:i:s');
+		return $s instanceof \DateTime ? $s->format('j. n. Y H:i:s') : $s;
```

Comment:
Úplně nejvís správně by zde měla být závislost na `\DateTimeInterface`, aby to příjmalo i `\DateTimeImmutable`